app/serialised_models.py

```diff
@@ -78,10 +78,10 @@ class SerialisedService(SerialisedModel):
         "name",
         "active",
         "contact_link",
+        # TODO: do we still need this?
         "normalised_service_name",
```

Comment:
i dont know if we need this at all - the only use of this now is to be a more robust uniqueness constraint than just a column default, and to set `email_sender_local_part` to when `custom_email_sender_name` is cleared, but that could be calculated on the spot

this could probs be entirely replaced by a custom check constraint that strips whitespace, lowercases, and does a few tweaks to normalise dashes, quotes, etc.